components/function-controller/internal/webhook/conversion_webhook_test.go

```diff
@@ -0,0 +1,389 @@
+package webhook
```

Comment:
Can you add use case when downgrading from v2 to v1 withouth repository name in annotation?